projects/explorer/src/app/pages/blocks/block/block.component.ts

```diff
@@ -28,6 +30,25 @@ export class BlockComponent implements OnInit {
         rest.tendermint.getBlockByHeight(sdk.rest, BigInt(height)).then((res) => res.data),
       ),
     );
+    const blockHash$ = this.block$.pipe(
+      map((block) => {
+        const base64Decoded = Uint8Array.from(Buffer.from(block.block_id?.hash!, 'base64'));
+        return Buffer.from(base64Decoded).toString('hex');
+      }),
+    );
+    const proposer$ = this.block$.pipe(
+      map((block) => {
+        console.log(block.block?.header?.proposer_address!);
+        const base64Decoded = Uint8Array.from(
+          Buffer.from(block.block?.header?.proposer_address!, 'base64'),
+        );
+        const publicKey = new proto.cosmos.crypto.ed25519.PubKey({
```

Comment:
How about

```suggestion
const proposer = new cosmosclient.ValAddress(base64Decoded);
```

Reply:
I left it neglected. I will check it out.